frontend/app/src/main/java/com/intersoft/groupup_app/navigation/EventDetailsPage.kt

```diff
@@ -0,0 +1,136 @@
+package com.intersoft.groupup_app.navigation
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.intersoft.auth.AuthContext
+import com.intersoft.event.EventManager
+import com.intersoft.ui.DisabledTextField
+import com.intersoft.ui.LabelText
+import com.intersoft.ui.ParticipantNumberDisplayField
+import com.intersoft.ui.PrimaryButton
+import com.intersoft.ui.TitleText
+import com.intersoft.utils.DateTimeManager
+
+@Composable
+fun EventDetailsPage(onGetEventFail: () -> Unit){
+    var eventName by remember {
+        mutableStateOf("")
+    }
+    var description by remember {
+        mutableStateOf("description")
+    }
+
+    var eventDate by remember {
+        mutableStateOf("")
+    }
+
+
+    var eventDuration by remember {
+        mutableIntStateOf(0)
+    }
+    var maxNumberOfParticipants by remember {
+        mutableStateOf(10)
+    }
+
+    var currentNumberOfParticipants = 0
+
+    var location by remember {
+        mutableStateOf("")
+    }
+
+    var host by remember {
+        mutableStateOf("")
+    }
+    var eventDataWasRecieved by remember{
+        mutableStateOf(false)
+    }
+
+     EventManager.getEvent(1,{onGetEventFail()}){
```

Comment:
the onGetEventFail parameter can be passed with no brackets, the line would look like this:
EventManager.getEvent(1,onGetEventFail){

Reply:
There are also merge conflicts that need to be resolved.